integration_test/common/testdata/integration-metadata_unique_expected-metrics-type/input.yaml

```diff
@@ -0,0 +1,117 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/activemq"
```

Comment:
Do we need such a large test file for this? And does it need real-world data? Should we pare it down to a few entries with generic names?

Reply:
It probably doesn't need real world data, but the whole file is validated at once so it has to pass all validations present in `IntegrationMetadata`

Reply:
Changed in: [a3b3e13](https://github.com/GoogleCloudPlatform/ops-agent/pull/669/commits/a3b3e13d3e634f380c64b51626a7316e97822d3e)

Reply:
Ok, this is no longer real world data, but do we need a file this large to test the validation? Wouldn't it be enough to have 2 (maybe 3) entries?

That said, it does seem to be consistent with the other tests, which also have overgrown test inputs, so maybe we can defer.

Reply:
I believe we should defer, I initially wanted to trim the test input but I thought it would be better for consistency.

Reply:
Changed in: [f9abb2d](https://github.com/GoogleCloudPlatform/ops-agent/pull/669/commits/f9abb2d6359abf815e008557f2c4c1a72669acf7)